Beta Release

tyler-dunkel opened the pull request:
New Services

* securityGroup
* securityGroupRule
* ccn
* ccnAttachment

Commit:
feat(kubernetesEngine): add tke service

Commit:
feat(securityGroup): add security group service

Commit:
feat(securityGroup): add security group service

Commit:
chore: add service alias

Commit:
chore: add service alias

james-zhou-inspire11 committed:
feat(routeTable): add routeTable

james-zhou-inspire11 committed:
feat(routeTable): add connection from routeTable to subnet

Commit:
Merge pull request #3 from cloudgraphdev/feature/CG-1122

Feature/CG-1122: add security group service

james-zhou-inspire11 committed:
feat(routeTable): update README

Commit:
feat(securityGroupRule): add securityGroupRule service

Commit:
feat(securityGroupRule): add securityGroupRule service

james-zhou-inspire11 committed:
feat(ccn): add ccn and ccn attachment

tyler-dunkel committed:
Merge pull request #5 from cloudgraphdev/feature/CG-1129

feat(securityGroupRule): add securityGroupRule service

Commit:
feat(networkAcl): add networkAcl service

james-zhou-inspire11 committed:
feat(ccn): update README

james-zhou-inspire11 committed:
feat(routeTable): update README

james-zhou-inspire11 committed:
feat(vpnGateway): add tencent vpnGateway

james-zhou-inspire11 committed:
feat(camPolicy): add tencent cam policy

Commit:
chore(readme): update supported services

tyler-dunkel committed:
Merge pull request #7 from cloudgraphdev/feature/CG-1127-tencent-ccn

Feature/cg 1127 tencent ccn

tyler-dunkel committed:
fix(services): fix ccn service connection file

tyler-dunkel committed:
fix(rebase): fastforward alpha to main

tyler-dunkel committed:
Merge pull request #8 from cloudgraphdev/feature/CG-1132-add-tencent-vpn-gateway

feat(vpnGateway): add tencent vpnGateway

tyler-dunkel committed:
chore(pipeline): add pipeline for github to tencent

tyler-dunkel committed:
Merge branch 'alpha' of github.com:cloudgraphdev/cloudgraph-provider-tencent into alpha

tyler-dunkel committed:
chore(pipeline): move .github folder

tyler-dunkel committed:
chore(pipeline): update npm scripts for build

autocloud-deploy-bot committed:
chore(release): 1.0.0-alpha.1

# 1.0.0-alpha.1 (2022-05-16)

### Bug Fixes

* **rebase:** fastforward alpha to main ([2a42aed](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/2a42aedd5a0488d49406b56d6b2eccffbb750aba))
* **services:** fix ccn service connection file ([ed87d28](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/ed87d280abef5619cc76357fcef54f3efa8706b3))
* initial release ([c8ee8f1](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/c8ee8f1ae4efd7357aa40453aa2960d1c63791f9))
* initial release ([911a0e7](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/911a0e7c580cc5d838c4a9adbeb677440e07d8ba))

### Features

* **boilerplate:** add boilerplate, vpc, subnet and tags ([a4b1da7](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/a4b1da78373b39deb24b0c02605c2cce0cac72a6))
* **ccn:** add ccn and ccn attachment ([f4ba707](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/f4ba70780e0d133094e209c8995b80027bd98bcf))
* **ccn:** update README ([0403c4f](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/0403c4fca7ea97270c499b17434e3c05b148ce72))
* **securityGroup:** add security group service ([1f20850](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/1f20850176f0e9f3c48e3f0d8fe86ac533370e7d))
* **securityGroupRule:** add securityGroupRule service ([a6e7144](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/a6e714495bb6669d338aa42fae43603eccc2bde0))
* **vpnGateway:** add tencent vpnGateway ([362bf7f](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/362bf7f37f9a858edabc233f655e19a62a5a7ca5))

james-zhou-inspire11 committed:
feat(routeTable): add routeTable service

james-zhou-inspire11 committed:
feat(vpnGatewayRoute): add vpnGatewayRoute service

james-zhou-inspire11 committed:
feat(vpnGatewayRoute): add vpnGatewayRoute service

james-zhou-inspire11 committed:
feat(camPolicy): add camPolicy service

tyler-dunkel committed:
Merge pull request #4 from cloudgraphdev/feature/CG-1117-tencent-route-table

Feature/cg 1117 tencent route table

autocloud-deploy-bot committed:
chore(release): 1.0.0-alpha.2

# [1.0.0-alpha.2](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/compare/1.0.0-alpha.1...1.0.0-alpha.2) (2022-05-18)

### Features

* **routeTable:** add connection from routeTable to subnet ([6a34907](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/6a3490754dc5da4aa3ebe68072b2f6fe5063130b))
* **routeTable:** add routeTable ([112f1d5](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/112f1d561208e122b36b51e842f2802cf1ee76be))
* **routeTable:** add routeTable service ([76ff05a](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/76ff05ac8cb7cad4ccdf68b5d74633248a350aa8))
* **routeTable:** update README ([f32ffec](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/f32ffecf3e67551826fae5025b3c2bf0e2a1feb4))
* **routeTable:** update README ([3fd9c90](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/3fd9c9080b6ee007ec598b86b108f301e34e41af))

james-zhou-inspire11 committed:
feat(customerGateway): add customerGateway service

tyler-dunkel committed:
Merge pull request #12 from cloudgraphdev/feature/CG-1126-add-vpn-customer-gateway-service

feat(customerGateway): add customerGateway service

autocloud-deploy-bot committed:
chore(release): 1.0.0-alpha.3

# [1.0.0-alpha.3](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/compare/1.0.0-alpha.2...1.0.0-alpha.3) (2022-05-19)

### Features

* **customerGateway:** add customerGateway service ([cdee2e8](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/cdee2e85baa41ce5364051641345af0563e2601f))

james-zhou-inspire11 committed:
feat(camUser): add camUser service

james-zhou-inspire11 committed:
feat(camUser): add camUser service

james-zhou-inspire11 committed:
feat(gatewayRoute): merge alpha

tyler-dunkel committed:
Merge pull request #11 from cloudgraphdev/feature/CG-1124-add-tencent-vpn-gateway-route

feat(vpnGatewayRoute): add vpnGatewayRoute service

autocloud-deploy-bot committed:
chore(release): 1.0.0-alpha.4

# [1.0.0-alpha.4](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/compare/1.0.0-alpha.3...1.0.0-alpha.4) (2022-05-24)

### Features

* **gatewayRoute:** merge alpha ([17d9777](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/17d9777c4a25f9b8ae6ea0799ee67e8a8281c8d7))
* **vpnGatewayRoute:** add vpnGatewayRoute service ([ba42cbe](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/ba42cbe2c892dd5ebfc826c899445f2c42aefc05))
* **vpnGatewayRoute:** add vpnGatewayRoute service ([f56783f](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/f56783f3f20b91bfe25af1dd959dd7ba4bccd0e6))

Commit:
Merge pull request #6 from cloudgraphdev/feature/CG-1130

feat(networkAcl): add networkAcl service

autocloud-deploy-bot committed:
chore(release): 1.0.0-alpha.5

# [1.0.0-alpha.5](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/compare/1.0.0-alpha.4...1.0.0-alpha.5) (2022-05-26)

### Features

* **networkAcl:** add networkAcl service ([3665d1e](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/3665d1e14782040d601152a252bd889e64fdef98))
* **securityGroup:** add security group service ([5be1232](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/5be123281dca7d37e29e91617f03ece3b6a2b07c))
* **securityGroupRule:** add securityGroupRule service ([c436eb8](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/c436eb8a15269f6738527a415421638f22e95193))

james-zhou-inspire11 committed:
feat(vpnConnection): add vpnConnection service

james-zhou-inspire11 committed:
feat(vpnConnection) add vpnConnection service

tyler-dunkel committed:
Merge pull request #14 from cloudgraphdev/feature/CG-1133-add-vpn-connection

feat(vpnConnection): add vpnConnection service

autocloud-deploy-bot committed:
chore(release): 1.0.0-alpha.6

# [1.0.0-alpha.6](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/compare/1.0.0-alpha.5...1.0.0-alpha.6) (2022-05-31)

### Features

* **vpnConnection:** add vpnConnection service ([f2b0072](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/f2b0072e237e787c2bf0d11358003a7a126e15de))

james-zhou-inspire11 committed:
feat(camPolicy): add camPolicy service

james-zhou-inspire11 committed:
feat(camPolicy): add camPolicy service

tyler-dunkel committed:
Merge pull request #10 from cloudgraphdev/feature/CG-1120-add-tencent-cam-policy

feat(camPolicy): add tencent cam policy

autocloud-deploy-bot committed:
chore(release): 1.0.0-alpha.7

# [1.0.0-alpha.7](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/compare/1.0.0-alpha.6...1.0.0-alpha.7) (2022-06-02)

### Features

* **camPolicy:** add camPolicy service ([3b8df16](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/3b8df166c3edd75f6b40f445f6a4375481d2f0ce))
* **camPolicy:** add camPolicy service ([3596551](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/35965518ed7f32da4bf024d780d56828c5250c60))
* **camPolicy:** add camPolicy service ([6c26c84](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/6c26c84f04399ce540f1c33a9e59bac3448def0c))
* **camPolicy:** add tencent cam policy ([123b8dd](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/123b8dd376242fdf07e167f1c10045f13d607c58))

james-zhou-inspire11 committed:
feat(camUser): add camUser service add region global

james-zhou-inspire11 committed:
feat(camUser): merge alpha

james-zhou-inspire11 committed:
feat(camUser): fix global region

tyler-dunkel committed:
Merge pull request #13 from cloudgraphdev/feature/CG-1123-tencent-cam-user

feat(camUser): add camUser service

autocloud-deploy-bot committed:
chore(release): 1.0.0-alpha.8

# [1.0.0-alpha.8](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/compare/1.0.0-alpha.7...1.0.0-alpha.8) (2022-06-06)

### Features

* **camUser:** add camUser service ([fa2c3b8](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/fa2c3b8dbe90781e28291f654734e6ee0b610e39))
* **camUser:** add camUser service ([caaaaa7](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/caaaaa7e548c146580a0758ed18865d3fc428545))
* **camUser:** add camUser service add region global ([c6162a6](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/c6162a69296e89aff6730f7fe706d137553a6d5f))
* **camUser:** fix global region ([feb5add](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/feb5add6bf7010952f3b81101e9a84c881a4af42))
* **camUser:** merge alpha ([c1ed872](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/c1ed8729c5afef905c0f04cf4ae01b87c1a595ef))

Commit:
fix(mutation): use schema names for mutations

Commit:
fix(mutation): use schema names for mutations

tyler-dunkel committed:
Merge pull request #2 from cloudgraphdev/feature/CG-1118

feat(kubernetesEngine): add tke service

autocloud-deploy-bot committed:
chore(release): 1.0.0-alpha.9

# [1.0.0-alpha.9](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/compare/1.0.0-alpha.8...1.0.0-alpha.9) (2022-06-07)

### Bug Fixes

* **mutation:** use schema names for mutations ([cbe8921](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/cbe8921a7c7c6f9b4dd3a52180c713853dc82ee0))
* **mutation:** use schema names for mutations ([3bec593](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/3bec59304a84c93176d9125bc7d0e527f88ff458))

### Features

* **kubernetesEngine:** add tke service ([ae097a7](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/ae097a7575c19611ede542feadbe381f8e233051))

csbrandt commented:
<img width="1064" alt="Screen Shot 2022-06-07 at 2 07 22 PM" src="https://user-images.githubusercontent.com/1036634/172483992-1cd090be-2416-46dc-9ae1-acd992852ec5.png">

james-zhou-inspire11 committed:
feat(camGroup): add camGroup service

james-zhou-inspire11 committed:
feat(camGroup): add camGroup service README

james-zhou-inspire11 committed:
fix(ccn): fix ccn and ccnAttachment

tyler-dunkel committed:
Merge pull request #16 from cloudgraphdev/fix/ccn-ccn-attachment

fix(ccn): fix ccn and ccnAttachment

autocloud-deploy-bot committed:
chore(release): 1.0.0-alpha.10

# [1.0.0-alpha.10](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/compare/1.0.0-alpha.9...1.0.0-alpha.10) (2022-06-10)

### Bug Fixes

* **ccn:** fix ccn and ccnAttachment ([384db27](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/384db271bb2b6d06ec2ab9bab6acf8d6b066effc))

tyler-dunkel committed:
Merge pull request #15 from cloudgraphdev/feature/CG-1131-add-tencent-cam-group

feat(camGroup): add camGroup service

autocloud-deploy-bot committed:
chore(release): 1.0.0-alpha.11

# [1.0.0-alpha.11](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/compare/1.0.0-alpha.10...1.0.0-alpha.11) (2022-06-10)

### Features

* **camGroup:** add camGroup service ([3c64503](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/3c645034eb189b0f6eccc59d61412602dcefaf29))
* **camGroup:** add camGroup service README ([86643b0](https://github.com/cloudgraphdev/cloudgraph-provider-tencent/commit/86643b08cb0e46e035f91b221bf4a80ff85ef08c))

csbrandt commented:
👍
